webapp/pages/profile/_id/_slug.vue

```diff
@@ -238,8 +238,10 @@
         <div
           v-if="hasMore"
           v-infinite-scroll="showMoreContributions"
-          infinite-scroll-disabled="$apollo.loading"
-          infinite-scroll-distance="10"
+          :infinite-scroll-immediate-check="true"
+          :infinite-scroll-disabled="$apollo.loading"
+          :infinite-scroll-distance="10"
+          :infinite-scroll-throttle-delay="800"
```

Comment:
default is 200ms... what went into the thinking to make it `800`? just your feeling? or was there something specific?

Reply:
It was try/error with @alina-beck. We tested it with seed data. Maybe we can increase it even further after we learned about the low performance of our neo4j pod.